Add podman integration tile (#11891)
DataDog/integrations-core: podman/README.md

# Podman Integration

[Podman][1] is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System. Containers can either be run as root or in rootless mode.

## Overview

> Podman container runtime is supported thanks to the [Agent Check: Container][2]

Reviewer: ```suggestion
Podman container runtime is supported by the [container Agent check][2].
```

> This check reports a set of metrics about any running containers, regardless of the runtime used to start them.

Reviewer: ```suggestion
This check reports a set of metrics on any running containers, regardless of the runtime used to start them.
```

> **NOTE**: The `container` check report standardized metrics for all containers found on the system, regardless of the container runtime.

Reviewer: ```suggestion
**NOTE**: The `container` check report standardizes metrics for all containers found on the system, regardless of the container runtime.
```

## Setup

### Installation

> Monitoring containers managed by [Podman][1], rely on the the [Agent Check: Container][2], its installation is describe [here][3].

Reviewer: ```suggestion
To monitor containers managed by [Podman][1], see the [installation instructions][3] for the [container Agent check][2].
```
